batch/batches/Import/KAsyncImport.class.php

```diff
@@ -205,7 +205,14 @@ private function fetchFile(KalturaBatchJob $job, KalturaImportJobData $data)
 
 		try
 		{
-			$sourceUrl = $data->srcFileUrl;
+            if ($data->shouldRedirect)
+            {
+                $sourceUrl =  ZoomHelper::getRedirectUrl($data->srcFileUrl, $data->urlHeaders);
```

Comment:
zoom is a plugin, you can access it like that from a generic import code.
if this function is independant on zoom need to have it in a general utilis class